modules/testcontainers/src/main/kotlin/jp/co/soramitsu/iroha2/testcontainers/IrohaContainer.kt

```diff
@@ -18,34 +11,33 @@ import java.nio.file.Files
 import java.nio.file.Path
 import java.time.Duration
 import java.util.UUID.randomUUID
-import java.util.function.Consumer
 import kotlin.io.path.absolute
 import kotlin.io.path.createTempFile
 
 open class IrohaContainer(
-    private val networkToJoin: Network = newNetwork(),
-    private val logConsumer: Consumer<OutputFrame>? = Slf4jLogConsumer(getLogger(IrohaContainer::class.java)),
     imageTag: String = DEFAULT_IMAGE_TAG,
-    private val genesis: Genesis = Genesis(RawGenesisBlock(mutableListOf())),
-    private val shouldCloseNetwork: Boolean = true,
+    config: IrohaConfig.() -> Unit = {}
 ) : GenericContainer<IrohaContainer>(DockerImageName.parse("$IMAGE_NAME:$imageTag")) {
 
+    private val config = IrohaConfig().apply(config)
+
     private val genesisFileLocation: Lazy<Path> = lazy { createTempFile("genesis-", randomUUID().toString()) }
 
     init {
-        this.withNetwork(networkToJoin)
+        this.withNetwork(this.config.networkToJoin)
             .withEnv(ENV_SUMERAGI_MAX_FAULTY_PEERS.first, ENV_SUMERAGI_MAX_FAULTY_PEERS.second)
             .withEnv(ENV_TORII_P2P_URL.first, ENV_TORII_P2P_URL.second)
             .withEnv(ENV_TORII_API_URL.first, ENV_TORII_API_URL.second)
             .withEnv(ENV_IROHA_PUBLIC_KEY.first, ENV_IROHA_PUBLIC_KEY.second)
             .withEnv(ENV_IROHA_ROOT_PUBLIC_KEY.first, ENV_IROHA_ROOT_PUBLIC_KEY.second)
             .withEnv(ENV_IROHA_PRIVATE_KEY.first, ENV_IROHA_PRIVATE_KEY.second)
             .withEnv(ENV_SUMERAGI_TRUSTED_PEERS.first, ENV_SUMERAGI_TRUSTED_PEERS.second)
+            .withEnv("MAX_LOG_LEVEL", this.config.maxLogLevel.name)
```

Comment:
Please declare it as constant with name starting with `ENV` prefix: `ENV_MAX_LOG_LEVEL` just like with other env variables